twentytwenty-blocks/twentytwenty-styles/editor-style-block.css

```diff
@@ -249,37 +254,37 @@ Inter variable font. Usage:
 .editor-styles-wrapper .wp-block h6 {
 	font-feature-settings: "lnum";
 	font-variant-numeric: lining-nums;
-	font-weight: 700;
+	font-weight: var(--wp--preset--font-weight--heading);
 	letter-spacing: -0.0415625em;
-	line-height: 1.25;
+	line-height: var(--wp--preset--line-height--heading);
 	margin: 40px 0 25px;
 }
 
 .editor-post-title__block .editor-post-title__input,
-.editor-styles-wrapper .wp-block h1 {
-	font-size: 36px;
-	font-weight: 800;
+.editor-styles-wrapper h1.wp-block {
+	font-size: 84px;
```

Comment:
Just wondering why we're using px values here when rem values are used in the frontend.

Reply:
In the frontend, Twenty Twenty [modifies the value of `1 rem`](https://github.com/WordPress/theme-experiments/blob/a442c52e37e6003eb66631383813fe2dcef1fffd/twentytwenty-blocks/twentytwenty-styles/style.css#L117) to be `10px` instead of the usual `13px`. Since that doesn't happen in the editor (and can't really, without messing with a lot of other font sizes), we're stuck with pixels for the moment. 

These should really be variables, but I didn't switch to using variables for these sizes because the heading font sizes don't match up with the small, normal, large, + huge sizes. I figured we'd keep things simple for now, and can open up a separate PR improve this.